content/handbook/sales/training/customer-success-skills-exchange.md

```
@@ -0,0 +1,76 @@
+title: "Technical Skills Exchange"
+description: "GitLab のテクニカルなフィールドチームメンバー向けの継続教育ウェブキャストシリーズ"
+upstream_path: /handbook/sales/training/customer-success-skills-exchange/
+upstream_sha: 1e195b58b9f249ff10bd0e705106c320fee86141
+translated_at: "2026-05-11T00:00:00Z"
+translator: claude
+stale: false
+---
+
+Technical Skills Exchange（旧称 Customer Success Skills Exchange）は、テクニカルなフィールドチームメンバー向けトピックに焦点を当てた隔週ウェビナーです。このシリーズは主に Customer Success と SA 組織を念頭に企画されていますが、フィールドチームメンバーであれば誰でも参加歓迎です。
+
+- 各トレーニングセッションには [スピーカー](#speaker-responsibilities) と [モデレーター](#moderator-responsibilities) がいます
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)